.travis.yml

```diff
@@ -5,9 +5,21 @@ language: python
 matrix:
     include:
         - python: 2.7
-          env: KERAS_BACKEND=tensorflow
+          env:
+            - KERAS_BACKEND=tensorflow
+            - TENSORFLOW_V=1.0.1
```

Comment:
I thought we were going for the last stable release (at the moment 1.1), and all past major versions (only 1.0 at the moment) ? This is what I meant here: https://groups.google.com/forum/#!topic/cleverhans-dev/TlyFojSkepU

Did you have something else in mind like all past major versions and all minor versions after the last major?